source/wp-content/themes/wporg-developer/inc/formatting.php

```diff
@@ -668,6 +669,40 @@ public static function fix_param_description_html_as_code( $text ) {
 		return $text;
 	}
 
+	/**
+	 * Wraps code-like references within 'code' tags.
+	 *
+	 * Example: https://developer.wordpress.org/reference/classes/wp_term_query/__construct/
+	 *
+	 * @param string $text Text.
+	 * @return string
+	 */
+	public static function fix_param_description_quotes_to_code( $text ) {
+		$textarr = preg_split( '/(<[^<>]+>)/', $text, -1, PREG_SPLIT_DELIM_CAPTURE ); // split out HTML tags
+		$text    = '';
+		foreach ( $textarr as $piece ) {
+			// HTML tags are untouched.
+			if ( str_starts_with( $piece, '<' ) ) {
+				$text .= $piece;
+				continue;
+			}
+
+			// Pipe delimited types inline.
+			$piece = preg_replace( "/(([\w'\[\]]+\|)+[\w'\[\]]+)/", '<code>$1</code>', $piece, -1 );
+
+			// Quoted strings.
+			$piece = preg_replace( "/('[^']{0,20}')/", '<code>$1</code>', $piece, -1 );
```

Comment:
I think the 20 character limit might be too short:

<img width="851" alt="Screen Shot 2022-06-08 at 11 32 33 AM" src="https://user-images.githubusercontent.com/541093/172659901-34142138-8ac5-4a88-a6ca-9451af8bb0c6.png">

Reply:
I originally chose 20 as something low to avoid mis-matching and generating a string like `Let<code>'s figure out a good length for the '</code>' quoting`. It would probably be better to just look for `''` without any spaces between them though perhaps?

Iteration is definitely needed here!

Reply:
[8028739](https://github.com/WordPress/wporg-developer/pull/97/commits/8028739bd2f085e69ca99ce72fd1a6820c38cb42)